docs/reference/installation.rst

```diff
@@ -282,7 +246,7 @@ Then, add a new custom firewall handlers for the admin:
                 pattern:            /admin(.*)
                 context:            user
                 form_login:
-                    provider:       fos_userbundle
+                    provider:       sonata_user_bundle
                     login_path:     /admin/login
```

Comment:
@jordisala1991 Question about Style:

login_path and the other routing parameters below support direct paths like "/admin/login" but they also support route ids like "sonata_user_admin_security_login"

what is better Style?

Reply:
We should use route name , I will change this too.

Reply:
Fixed.